playbooks/upgrade-zuul.yaml

```diff
@@ -63,7 +63,7 @@
         # npm pack
         # npm run  build:dist
       args:
-        chdir: '{{ ansible_user_dir }}/src/git.openstack.org/openstack-infra/zuul'
+        chdir: '{{ ansible_user_dir }}/src/opendev.org/openstack-infra/zuul'
```

Comment:
openstack-infra

Reply:
No, zuul is under the https://github.com/openstack-infra/zuul

Reply:
NOTE: The openinfra team only put some openstack related repo into openstack's repo, but there are still some projects under the openstack-infra repo.